### PR TITLE
Yuqxia/validateop

### PR DESCRIPTION
**Pull request by xiayuqing0622**
support xbox op: cast,ceil,clip,concat,constant,constantofshape,conv,convtranspose,cos,cumsum,depthtospace,div,sqrt,sub,tan,tanh,tile,tanspose,where.
note:
1. to test constant and constantofshape op, need to comment line495-500. https://github.com/microsoft/nnfusion/blob/3b6db3e25eed14fc530247fc68715c14a22a8ea4/src/nnfusion/engine/pass/codegen/cuda_codegen_pass.cpp#L495-L500
2. test_transpose_all_permutations_0 will fail because it is memcpy

### CODE REVIEW

File: src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp [hunk 1]
```diff
@@ -44,19 +44,29 @@ namespace nnfusion
...
-                    NNFUSION_CHECK(nnfusion::shape_size(value.get_shape()) == 1);
-                    auto const_op =
-                        make_constant_op(value.get_ng_type(),
-                                         Shape(std::begin(output_shape), std::end(output_shape)),
-                                         value);
+                    std::shared_ptr<nnfusion::op::Constant> const_op;
+                    NNFUSION_LOG(INFO) << Shape(std::begin(output_shape), std::end(output_shape));
+                    try
+                    {
+                        auto value = node.get_attribute_value<Tensor>("value");
```
Comment (mzmssg) [inline src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp:51]: try catch is for? Using node.has_attribute is enough?

File: src/nnfusion/frontend/util/evaluator.hpp [hunk 2]
```diff
@@ -20,7 +20,7 @@ namespace nnfusion
             std::vector<std::vector<char>>
                 get_node_outputs(std::shared_ptr<GNode> gnode, int depth = 0, int arg_idx = 0)
             {
-                // NNFUSION_CHECK(gnode->get_op_type() != "Parameter");
+                NNFUSION_CHECK(gnode->get_op_type() != "Parameter");
                 if (gnode->get_op_type() == "Parameter")
```
Comment (mzmssg) [inline src/nnfusion/frontend/util/evaluator.hpp:24]: remove this if?

File: src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp (hunk 1 above)
Comment (xiayuqing0622) [reply src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp:51]: thanks for pointing it out, node.has_attribute() is better

File: src/nnfusion/frontend/util/evaluator.hpp (hunk 2 above)
Comment (xiayuqing0622) [reply src/nnfusion/frontend/util/evaluator.hpp:24]: ok

Comment (xiayuqing0622) [reply src/nnfusion/frontend/util/evaluator.hpp:24]: > remove this if?

ok

File: src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp (hunk 1 above)
Comment (xiayuqing0622) [reply src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp:51]: > try catch is for? Using node.has_attribute is enough?

thanks for pointing it out, node.has_attribute() is better

**Review by mzmssg — APPROVED** [review]

**Pull request merged**